### PR TITLE
97978 Get Events-Homepage Selection to Persist in Admin and Customizer

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#97978**](http://central.tri.be/issues/97978)

**Code Review Note:** A readme entry already exists for this ticket; this is QA follow-up dev.

### CODE REVIEW

File: src/Tribe/Customizer/Front_Page_View.php [hunk 1]
```diff
@@ -45,14 +45,27 @@ public function setup() {
 	 */
 	public function add_events_page_option( $output, $r, $pages ) {
```
Comment (bordoni) [inline src/Tribe/Customizer/Front_Page_View.php]: Can we use a better variable name for `$r`?

Comment (ggwicz) [reply src/Tribe/Customizer/Front_Page_View.php]: that's the name of the var in the core function so i just kept it consistent, which i think is the right move -- thoughts?

Comment (bordoni) [reply src/Tribe/Customizer/Front_Page_View.php]: Change it to something more read-able if we can.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $pages is never used | [Front_Page_View.php:46](https://github.com/moderntribe/the-events-calendar/blob/13b64e8e00c533bc051206261188993c8211b266/src/Tribe/Customizer/Front_Page_View.php#L46)

> via jenkins/codesniffer-phpcs

**Pull request merged**